charts/ceph-csi-cephfs/README.md

```diff
@@ -118,6 +118,7 @@ charts and their default values.
 | `commonLabels`                                 | Labels to apply to all resources                                                               | `{}`                                                  |
 | `logLevel`                                     | Set logging level for csi containers. Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.                          | `5`                                                |
 | `sidecarLogLevel`                              | Set logging level for csi sidecar containers. Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.                  | `1`                                                |
+| `logSlowOperationsAfter`                       | Operations running longer than the specified time duration will be logged as slow. Setting the value to zero disables the feature.                   | `15s`                                              |
```

Comment:
can we keep this value as same as GRPC timeout which we have at `60s` seconds?

Reply:
Having the same timeout probably defeats the purpose? It would be useful to know if an operation is still running before the timeout hits.